diff --git a/deps/v8/include/v8-object.h b/deps/v8/include/v8-object.h
index 8e46e71..3c74d12 100644
--- a/deps/v8/include/v8-object.h
+++ b/deps/v8/include/v8-object.h
@@ -562,7 +562,7 @@ class V8_EXPORT Object : public Value {
   V8_INLINE static void* GetAlignedPointerFromInternalField(
       const PersistentBase<Object>& object, int index) {
     return object.template value<Object>()->GetAlignedPointerFromInternalField(
-        index);
+        index, kEmbedderDataTypeTagDefault);
   }
 
   /** Same as above, but works for TracedReference. */
@@ -579,7 +579,7 @@ class V8_EXPORT Object : public Value {
   V8_INLINE static void* GetAlignedPointerFromInternalField(
       const BasicTracedReference<Object>& object, int index) {
     return object.template value<Object>()->GetAlignedPointerFromInternalField(
-        index);
+        index, kEmbedderDataTypeTagDefault);
   }
 
   /**
diff --git a/deps/v8/include/v8-profiler.h b/deps/v8/include/v8-profiler.h
index 7e2d093..2de2619 100644
--- a/deps/v8/include/v8-profiler.h
+++ b/deps/v8/include/v8-profiler.h
@@ -1132,7 +1132,6 @@ class V8_EXPORT HeapProfiler {
     /**
      * The resolver used by the snapshot generator to get names for V8 objects.
      */
-    V8_DEPRECATED("Use context_name_resolver callback instead.")
     ObjectNameResolver* global_object_name_resolver = nullptr;
     /**
      * The resolver used by the snapshot generator to get names for v8::Context
